CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.12.1 (July 04, 2022)
+
+BUG FIXES:
+
+* template functions: funtion `title` creates capitalize strings instead of upper strings ([#165](https://github.com/hashicorp/terraform-plugin-docs/pull/165)).
+
```

Comment:
Another PR landed so I think you need to rebase and put this in the `0.13.0` section, but still under `BUG FIXES`.